manifests/components/kibana.jsonnet

```diff
@@ -38,18 +39,69 @@ local strip_trailing_slash(s) = (
     },
   },
 
+  // List of Kibana plug-ins to install
+  plugins:: {
+    /* Example:
+    logtrail: {
+      version: "v0.1.31",
+      url: "https://github.com/sivasamyk/logtrail/releases/download/v0.1.31/logtrail-6.7.1-0.1.31.zip",
+    },
+    */
+  },
+
   es: error "elasticsearch is required",
 
   serviceAccount: kube.ServiceAccount($.p + "kibana") + $.metadata {
   },
 
+  pvc: kube.PersistentVolumeClaim($.p + "kibana-plugins") + $.metadata {
+    storage: "15Gi",
+  },
+
   deploy: kube.Deployment($.p + "kibana") + $.metadata {
     spec+: {
       template+: {
         spec+: {
           securityContext: {
             fsGroup: 1001,
           },
+          volumes_+: {
+            plugins: kube.PersistentVolumeClaimVolume($.pvc),
+          },
+          initContainers_+: {
+            elasticsearch_plugins_install: kube.Container("elasticsearch-plugins-install") {
```

Comment:
shouldn't this name be `kibana_plugins_install`?
 (just a small suggesstion: I would rename it like `kibana-plugin-installer`)

Reply:
Indeed!